src/types/User.ts

```diff
@@ -57,6 +61,11 @@ export interface ImportedUser {
   importedOn?: Date;
   status?: ImportStatus
   errorDescription?: string;
+  importedData?: {
+    autoActivateUserAtImport: boolean;
+    autoActivateTagAtImport: boolean;
```

Comment:
Same here: We cannot have Tags when importing users, right?

Reply:
done ✅